packages/fast-components-styles-msft/src/index.ts

```diff
@@ -5,6 +5,10 @@ import ActionTriggerStyles from "./action-trigger";
 export { ActionTriggerStyles };
 export * from "./action-trigger";
 
+import BadgeStyles from "./badge";
+export { BadgeStyles };
+export * from "./badge";

```

Comment:
I don't think that the export * is exporting anything additional here.

Reply:
In the MSFT styles package? I'm not seeing those as being imported.

Reply:
Sorry, wrong package. Fixed.